projects/or_scs/src/Mirza/SupplyChain/PopulateUtils.hs

```diff
@@ -129,8 +129,9 @@ insertAndAuth scsUrl orUrl authToken locMap ht (entity:entities) = do
 
 insertEachEvent :: EachEvent ->  ClientM ()
 insertEachEvent (EachEvent [] _) = pure ()
-insertEachEvent (EachEvent _ ev) = do
-  void $ SCSClient.insertGS1Event ev
+insertEachEvent (EachEvent _entities ev) = do
```

Comment:
The point here was that this code was just commented out because it relied on the BR and now that we've moved to auth zero we have no workflow atm for including authing from this test case, hopefully at some point we might complete a fix here and so was just trying to preserve the code as close to the original (i.e. matching unused variable names with the section that two lines down thats commented out) so that its easy to uncomment and reinclude once we have a work around. The effort is probably futile, and yeah either way is fine.